book/entities.rst

```diff
@@ -0,0 +1,1182 @@
+    |                   | procedure.                                                              |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``indexed``       | Set this to ``true`` when the attribute needs to be accessed in SQL     |
+    |                   | queries (see :ref:`book-entities-indexed-attributes`).                  |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``priority``      | Defines the order in which options will be shown in grid views and      |
+    |                   | forms (options with a higher priority will be displayed before options  |
+    |                   | with a lower priority).                                                 |
+    +-------------------+-------------------------------------------------------------------------+
+
+``grid``
+    Configures the way the field is presented in a list view:
+
+    +-------------------+-------------------------------------------------------------------------+
+    | Option            | Description                                                             |
+    +===================+=========================================================================+
+    | ``type``          | The attribute type                                                      |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``label``         | The grid column headline                                                |
+    +-------------------+-------------------------------------------------------------------------+
+    | * ``show_filter`` | These options control whether the view can be filtered by the attribute |
+    | * ``filterable``  | value and how the filter options look like.                             |
+    | * ``filter_type`` |                                                                         |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``sortable``      | When enabled, the user can sort the table by clicking on the attribute  |
+    |                   | column's title.                                                         |
+    +-------------------+-------------------------------------------------------------------------+
+
+    .. note::
+
+        In order to use the attribute in a grid view, it
+        :ref:`needs to be indexed <book-entities-indexed-attributes>`.
+
+``form``
+    You use these options to control how the actual value can be configured by the user:
+
+    =================  ==============================================================
+    Option             Description
+    =================  ==============================================================
+    ``type``           The form type
+    ``options``        Additional options controlling the form layout:
```

Comment:
actually the list of available options depend on the form type specified in the `type` option. For example if you change the form type from `choice` to the `text` the `choices` and `empty_value` form options will not be available